src/System.Private.ServiceModel/tests/Scenarios/Binding/Custom/CustomBindingTests.4.0.0.cs

```diff
@@ -2,17 +2,20 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-
-using Infrastructure.Common;
 using System;
 using System.ServiceModel;
 using System.ServiceModel.Channels;
+using Infrastructure.Common;
 using Xunit;
 
 public partial class CustomBindingTests : ConditionalWcfTest
 {
     // Http: Client and Server bindings setup exactly the same using default settings.
+#if FULLXUNIT_NOTSUPPORTED
     [Fact]
+#endif
+
+    [WcfFact]
```

Comment:
Reviewers: to help understand the pattern, WcfFact, Condition, and Issue attributes are all defined in ToF (TFS only) so that they compile (but are non-functional).  This permits everything outside the #if to be correct for both GitHub and ToF.  The goal is to be able to remove all the 'if FULLXUNIT_NOTSUPPORTED' blocks when we are on Helix, and the rest should not need to be touched.